activerecord/lib/active_record/persistence.rb

```diff
@@ -102,21 +102,21 @@ def persisted?
 
     # Saves the model.
     #
-    # If the model is new a record gets created in the database, otherwise
+    # If the model is new, a record gets created in the database, otherwise
     # the existing record gets updated.
     #
-    # By default, save always run validations. If any of them fail the action
+    # By default, +save+ always runs validations. If any of them fail the action
     # is cancelled and +save+ returns +false+. However, if you supply
     # validate: false, validations are bypassed altogether. See
     # ActiveRecord::Validations for more information.
     #
-    # By default, #save also sets the +updated_at+/+updated_on+ attributes to
+    # By default, +#save+ also sets the +updated_at+/+updated_on+ attributes to
     # the current time. However, if you supply <tt>touch: false</tt>, these
```

Comment:
@claudiob You had changed `+save+` to `#save` in bb78af73. However, at many other places method names are used as `+method_name+` and even `+#method_name+`.

Is there a preferred syntax?